.devcontainer/scripts/setup.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+pip install .
+python setup.py build
+
+# sudo mkdir /usr/local/share/jupyter
+# sudo chown vscode /usr/local/share/jupyter
+# jupyter labextension develop . --overwrite --no-build
+jupyter server extension enable nbdime # if developing for jupyter lab or nbclassic
+
+jupyter nbextension install --py nbdime --sys-prefix [--sym-link]
+jupyter nbextension enable --py nbdime --sys-prefix
+
+jupyter labextension link ./packages/nbdime --no-build
+jupyter labextension install ./packages/labextension
```

Comment:
@vidartf This codespace now includes Jupyter Lab, this will allow people starting up the codespace to have a pre-install Jupyter Lab to develop against. 

However I am not sure the best way to link the local version of nbdime to the JupyterLab server.

This works, but I am not sure how to get changes to show up, I need to rebuild the docker container every time which is not great.

Is there some sort of script I could put together to automatically update Jupyter Lab? Thanks..

Reply:
I was trying to use this command

`jupyter labextension develop --overwrite .`

But it was not working as I would expect. I wonder if there is some prereq I am not aware of that the cookie cutter does?

Reply:
Cool thanks. I will dig into this